tests/tx.test.js

```diff
@@ -21,14 +21,14 @@ describe('Full flow', () => {
     const tokens = tokensResponse.tokens
 
     // Deposit. tokens[0] is Eth, tokens[1] is an ERC20
-    const depositTokenParams = await Tx.deposit(depositAmount, account.hermezEthereumAddress,
+    const depositTokenTxData = await Tx.deposit(depositAmount, account.hermezEthereumAddress,
       tokens[1], account.hermezWallet.publicKeyCompressedHex, 'http://localhost:8545')
-    expect(depositTokenParams).toEqual([`0x${account.hermezWallet.publicKeyCompressedHex}`, 0, 37864,
-      0, tokens[1].id, 0, '0x'])
-    const depositEthParams = await Tx.deposit(depositEthAmount, account.hermezEthereumAddress,
+    expect(depositTokenTxData).toBeTruthy()
```

Comment:
Aren't the returned values repeatable enough to be able to test something more precise?